core/src/main/java/org/elasticsearch/index/query/SpanMultiTermQueryBuilder.java

```diff
@@ -62,11 +65,26 @@ protected void doXContent(XContentBuilder builder, Params params)
     @Override
     protected Query doToQuery(QueryShardContext context) throws IOException {
         Query subQuery = multiTermQueryBuilder.toQuery(context);
+        float boost = AbstractQueryBuilder.DEFAULT_BOOST;
+        if (subQuery instanceof BoostQuery) {
+            BoostQuery boostQuery = (BoostQuery) subQuery;
+            subQuery = boostQuery.getQuery();
+            boost = boostQuery.getBoost();
+        } else if (subQuery instanceof SpanBoostQuery) {
+            //this should never happen as MultiTermQuery extends SpanQuery, but who knows...
```

Comment:
yea I meant as none of the multi term queries in lucene extend SpanQuery . Shall we simply drop this branch then?